src/parser/class.js

```diff
@@ -201,22 +201,18 @@ module.exports = {
         const name = this.text().substring(1); // ignore $
         this.next();
         propName = propName(name);
+
+        let value = null;
+
         if (this.token === ";" || this.token === ",") {
-          return result(propName, null, readonly, nullable, type, attrs || []);
+          // no-op
         } else if (this.token === "=") {
           // https://github.com/php/php-src/blob/master/Zend/zend_language_parser.y#L815
-          return result(
-            propName,
-            this.next().read_expr(),
-            readonly,
-            nullable,
-            type,
-            attrs || [],
-          );
+          value = this.next().read_expr();
         } else {
           this.expect([",", ";", "="]);
```

Comment:
I think we need to drop the `"="` in th eexpect, we already handled that with the previous if condition.
```suggestion
          this.expect([",", ";"]);
```

Reply:
"," and ";" are handled by the first if.
I think this line is here to show an error message indicating that we expect one of those 3 things. 

I figured this out while working on adding support for property hooks.